queries.sql

```diff
@@ -117,4 +117,62 @@ WHERE O.full_name = 'Dean Winchester' AND escape_attempts = 0;
 SELECT COUNT(*) as count, full_name FROM animals A
 INNER JOIN owners O ON A.owner_id = O.id
 GROUP BY O.full_name
-ORDER BY count DESC;
\ No newline at end of file
+ORDER BY count DESC;
+
+
```

Comment:
![Capture0](https://user-images.githubusercontent.com/93039359/206868746-48846388-6a8a-4fdb-80ac-4da62de213b2.PNG)

- [ ] It's great that you have added a descriptive readme but i guess this link is not working 😨 It's because you don't hav e a license file in your repository 🤔 It would be nice if you add a license file using these [guidlines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) 👍

Reply:
Thank you. I resolved the issue